scripts/partition_manager/partition_manager.rst

```diff
@@ -189,11 +190,12 @@ share_size: list
    This property can be set for image or placeholder partitions.
    It cannot be used by phony partitions.
    The list can contain any kind of partition.
+   ``share_size`` takes precedence over ``size`` if one or more partitions in ``share_size`` exists.
 
    If the target partition is the ``app`` or a partition that spans over the ``app``, the size is effectively split between them, because the size of the ``app`` is dynamically decided.
 
-   If non of the partitions in the ``share_size`` list exists, and the partition does not define a ``size`` property, then the partition is removed.
-   If non of the partitions in the ``share_size`` list exists, and the partition **does** define a ``size`` property, then the ``size`` property is used to set the size.
+   If none of the partitions in the ``share_size`` list exists, and the partition does not define a ``size`` property, then the partition is removed.
+   If none of the partitions in the ``share_size`` list exists, and the partition **does** define a ``size`` property, then the ``size`` property is used to set the size.
```

Comment:
The history should be re-written to avoid this patch.

Reply:
will do